packages/inter-protocol/test/test-priceAggregatorChainlink.js

```diff
@@ -618,7 +618,8 @@ test('suggest', async t => {
 
   t.deepEqual(oracleBSuggestion.eligibleForSpecificRound, false);
   t.deepEqual(oracleBSuggestion.queriedRoundId, 2n);
-  t.deepEqual(oracleBSuggestion.oracleCount, 3);
+  // FIXME do we need oracleCount?
+  // t.deepEqual(oracleBSuggestion.oracleCount, 3);
```

Comment:
@michaelfig do you know any requirements for this? It seems like a holdover from https://github.com/smartcontractkit/chainlink/blob/026e9a69dbcb057123264392e1e5a0c2c03e96f0/contracts/src/v0.6/FluxAggregator.sol#L321-L326

Reply:
Is it redundant? i.e. is there a list of oracles where you can just look at the `.length`?

The design in #4639 called for "Source information (per oracle node operator)".

Reply:
yeah, the fix was pretty simple, making the contract add it by counting its oracles. d7674a0b1127a249858f8b00e0c0ddb95bd391fd

While cleaning that up I noticed that `startingRound` and `endingRound` were intended to be used for enabling and disabling an oracle. Since we do that through governance I've removed them in fcdb451d15293d6f99df47ccf0124f077789091a cc @michaelfig